### PR TITLE
[5.0] Support before/after filter specification in Route::resource

### PR DESCRIPTION
**Pull request by tomzx**

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Shouldn't we use array_merge?

**Comment by tomzx** [comment]
Why should we? The array union does what is needed to make it work. Maybe I'm missing something?

File: tests/Routing/RoutingRouteTest.php [hunk 1]
```diff
@@ -909,3 +925,12 @@ public function handle()
...
 }
+
+class RouteResourceControllerStub {
+	public function index(){}
+	public function show(){}
+	public function create(){}
+	public function edit(){}
+	public function update(){}
+	public function delete(){}
+}
```
Comment (GrahamCampbell) [inline tests/Routing/RoutingRouteTest.php]: Missing new line character.

Comment (tomzx) [inline tests/Routing/RoutingRouteTest.php]: Fixed!

**Comment by taylorotwell** [comment]
Would rather them just be specified on the controller.

**Pull request closed without merging**